src/views/TeamView.vue

```diff
@@ -0,0 +1,117 @@
+<template>
+  <div>
+      <h1>{{ name }}  <img v-if="permission >= 2" src="../../assets/edit-icon.svg" alt="edit"></h1>
```

Comment:
Can we use constants for permissions? We probably should have started that earlier on...

Reply:
Agreed!